tests/transforms/core/test_transform_program.py

```diff
@@ -1394,6 +1421,176 @@ def f():
         ):
             assert eqn.primitive == expected_primitive
 
+    def test_call_fallback_on_qnode(self):
+        """Test that a TransformProgram can be applied to a QNode using the fallback."""
+
+        program = TransformProgram()
+        program.add_transform(qml.transforms.cancel_inverses)
+        program.add_transform(transform(first_valid_transform), 0)
```

Comment:
Could actually start doing:


```suggestion
        program += qml.transforms.cancel_inverses
        program += transform(first_valid_transform)(0)
```